examples/jaeger-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,132 @@
+};
+
+{
+  grafanaDashboards+: {
+    'jaeger-write.json':
+      g.dashboard('Jaeger / Write')
+      .addRow(
+        g.row('Services')
+        .addPanel(
+          g.panel('span creation rate') +
+          g.qpsPanelErrTotal('jaeger_reporter_spans{result=~"dropped|err"}', 'jaeger_reporter_spans') +
+          g.stack
+        )
+        .addPanel(
+          g.panel('% spans dropped') +
+          g.queryPanel('sum(rate(jaeger_reporter_spans{result=~"dropped|err"}[1m])) by (namespace) / sum(rate(jaeger_reporter_spans[1m])) by (namespace)', '{{namespace}}') +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) } +
+          g.stack
+        )
+      )
+      .addRow(
+        g.row('Agent')
+        .addPanel(
+          g.panel('batch ingest rate') +
+          g.qpsPanelErrTotal('jaeger_agent_reporter_batches_failures_total', 'jaeger_agent_reporter_batches_submitted_total') +
+          g.stack
+        )
+        .addPanel(
+          g.panel('% batches dropped') +
+          g.queryPanel('sum(rate(jaeger_agent_reporter_batches_failures_total[1m])) by (cluster) / sum(rate(jaeger_agent_reporter_batches_submitted_total[1m])) by (cluster)', '{{cluster}}') +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) } +
+          g.stack
+        )
+      )
+      .addRow(
+        g.row('Collector')
+        .addPanel(
+          g.panel('span ingest rate') +
+          g.qpsPanelErrTotal('jaeger_collector_spans_dropped_total', 'jaeger_collector_spans_received_total') +
+          g.stack
+        )
+        .addPanel(
+          g.panel('% spans dropped') +
+          g.queryPanel('sum(rate(jaeger_collector_spans_dropped_total[1m])) by (instance) / sum(rate(jaeger_collector_spans_received_total[1m])) by (instance)', '{{instance}}') +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) } +
+          g.stack
+        )
+      )
+      .addRow(
+        g.row('Collector - Queue Stats')
+        .addPanel(
+          g.panel('span queue length') +
+          g.queryPanel('jaeger_collector_queue_length', '{{instance}}') +
+          g.stack
+        )
+        .addPanel(
+          g.panel('span queue time - 95 percentile') +
+          g.queryPanel('histogram_quantile(0.95, sum(rate(jaeger_collector_in_queue_latency_bucket[1m])) by (le, instance))', '{{instance}}')
+        )
+      )
+      .addRow(
+        g.row('Cassandra')
+        .addPanel(
+          g.panel('insert attempt rate') +
+          g.qpsPanelErrTotal('jaeger_cassandra_errors_total', 'jaeger_cassandra_attempts_total') +
+          g.stack
+        )
+        .addPanel(
+          g.panel('% inserts erroring') +
+          g.queryPanel('sum(rate(jaeger_cassandra_errors_total[1m])) by (instance) / sum(rate(jaeger_cassandra_attempts_total[1m])) by (instance)', '{{instance}}') +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) } +
+          g.stack,
+        )
+      ),
+
+    'jaeger-read.json':
+      g.dashboard('Jaeger / Read')
```

Comment:
I'm not sure this deserves its own dashboard. Are you using it at Grafana? Perhaps it would be more appropriate to have another row in the previous dashboard for the Jaeger Query?